src/openrct2/world/Climate.h

```diff
@@ -9,8 +9,12 @@
 
 #pragma once
 
+#include "../core/EnumUtils.hpp"
+
+#include <array>
 #include <cstdint>
 
+// TODO: refactor usage and remove
```

Comment:
Before merging the PR, or?

Reply:
Yes, any remaining usage should be replaced with `RCT12::ClimateType`. I think we're there, but I'll have to check.

Reply:
Ah right, removal is pending the purge of `ClimateSetAction`. I've left that for last, because it requires updating the replays. Before that's done, I'd like to ensure there are no regressions.